__tests__/endpoint.tests/endpoint.tests.js

```diff
@@ -85,4 +85,56 @@ describe("articles", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+
+  test("GET:200 sends an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.articles)).toBe(true);
+        expect(body.articles.length).toBe(5);
+      });
+  });
+
+  test("GET:200 returns article with comment count", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles[0]).toMatchObject({
+          article_id: 3,
+          author: "icellusedkars",
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
+        });
+      });
+  });
```

Comment:
You could be more thorough in checking the response data types by checking the length of the returned array of articles and using a forEach to assert that all the properties are of the correct data type. For example:

`    author: expect.any(String),
  title: expect.any(String),
`

 Let me know if you need more info on this 👍